src/Module.php

```diff
@@ -124,7 +124,7 @@ public function getConfig()
         return include __DIR__ . '/../config/module.config.php';
     }
 
-    public function getViewHelperConfig()
+    public function getViewHelperConfig(): array
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!